src/demos/popover.html

```diff
@@ -210,5 +211,29 @@ <h1 style="margin: 0 auto; text-align: center">Popover</h1>
         </calcite-tabs>
       </div>
     </div>
+
+    <!-- virtual el popover -->
+    <calcite-popover id="virtual-popover" open label="right start popover" placement="right" id="popover-right" open>
```

Comment:
Can this be moved to a usage file? I think it'd be helpful to show the different ways to set up the reference element. This also applies to the tooltip.

We can tackle this in a separate doc issue. cc @benelan

Reply:
I'll move to usage file 👍